test/Makefile

```diff
@@ -230,13 +241,14 @@ stop-approve-matic: ## Stops approve in node container
 
 .PHONY: run
 run: ## Runs a full node
+	$(RUNGRAFANA)
```

Comment:
@xavier-romero in this `run` recipe we don't have the `RUNMETRICS` so we don't launch the Prometheus scraper.
The idea was, since metrics are disabled by default, to launch it separately using `run-metrics` only when they are enabled.
IMHO we could do the same for Grafana (I see you added the `run-grafana`), or maybe have a single recipe to start both Prometheus and Grafana, WDYT?

Reply:
@kind84 This setup does not really put the focus on prometheus endpoint, since there is only one available yet, but sets up the components needed to grab it when ready, and also grabs basic docker info like cpu, memory, net, disk, that can be usefull (I guess!) when running tests or metrics. From this point, you can easily add more prometheus endpoints to grab info from when available.
So, where/when does it fits best to set these containers up I don't really know, please feel free to set up where you as a node team feels it best suits. I just pushed the skeleton, I expect you to adjust as per your requeriments.

Reply:
```suggestion
```

@xavier-romero, discussing with @arnaubennassar, we feel it's better to remove this line from `make run` and use `make run-grafana` when explicitly want to enable it.

I remind you that `make run` is the command that we normally use in development and we probably don't need grafana most of the time.

We can discuss about having another Makefile recipe for a "deployment" node including our custom metrics, Prometheus and Grafana.